src/features/user/MFV2/ProfileCard/PrivateProfileCard/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { ProfileProps } from '../../../../common/types/profile';
+import { Avatar } from '@mui/material';
+import getImageUrl from '../../../../../utils/getImageURL';
+import styles from './PrivateProfileCard.module.scss';
+import {
+  DefaultUserProfileImage,
+  SettingsIcon,
+} from '../../../../../../public/assets/images/icons/ProfilePageV2Icons';
+import { useLocale, useTranslations } from 'next-intl';
+import ProfileCardButtonContainer from '../ProfileCardButton/ProfileCardButtonContainer';
+import { useRouter } from 'next/router';
+
+const PrivateProfileCard = ({ userProfile }: ProfileProps) => {
+  const t = useTranslations('Profile');
+  const screenWidth = window.innerWidth;
+  const isMobile = screenWidth <= 767;
+  const router = useRouter();
+  const locale = useLocale();
+  const isPublicAccount = router.asPath === `/${locale}/profile/mfv2-public`;
+
+  return (
+    <div className={styles.profileCardContainer}>
+      <div className={styles.profileBackground}></div>
+      <div className={styles.profilePicture}>
+        {userProfile?.image ? (
+          <Avatar
+            alt="user Image"
+            src={getImageUrl('profile', 'avatar', userProfile?.image)}
+            className={styles.avatar}
+          />
+        ) : (
+          <DefaultUserProfileImage />
+        )}
+      </div>
+      <div className={styles.profileDetailsContainer}>
+        {!isPublicAccount && (
+          <button className={styles.editProfileIcon}>
+            <SettingsIcon />
+          </button>
```

Comment:
Since this is going to take the user to the Edit Profile page, it should be a link. If functionality changes to a modal, we can update later.